torchx/schedulers/kueue_job_scheduler.py

```diff
@@ -0,0 +1,820 @@
+#!/usr/bin/env python3
```

Comment:
do we need to keep this file? seems like kueue_scheduler is the same?

Reply:
That is odd I don't know how that snuck in there I will remove :+1: